pkg/virt-handler/device-manager/device_controller_test.go

```diff
@@ -104,7 +100,7 @@ var _ = Describe("Device Controller", func() {
 
 	Context("Basic Tests", func() {
 		It("Should indicate if node has device", func() {
-			deviceController := NewDeviceController(host, 10, "rw", fakeConfigMap, clientTest.CoreV1())
+			deviceController := NewDeviceController(host, nil, fakeConfigMap, clientTest.CoreV1())
```

Comment:
I think it would be more readable if you used an helper variable here; nil says too little.

```golang
var noDevices []GenericDevice
deviceController := NewDeviceController(host, noDevices, fakeConfigMap, clientTest.CoreV1())
...
```

Reply:
I disagree; this clearly shows an empty slice of `GenericDevice`s - thus I know both the type of the object, and that the slice is empty.

Reply:
Ok, changed.